protocol/message.go

```diff
@@ -46,5 +46,16 @@ type KeyLookupRequest struct {
 type KeyLookupProof struct {
 	Type int
 	AP   *merkletree.AuthenticationPath
-	TB   *merkletree.TemporaryBinding
+	TB   *merkletree.TemporaryBinding `json:",omitempty"`
+}
+
+type ConsistencyCheckRequest struct {
+	Username string `json:"username"`
+	Epoch    uint64 `json:"epoch"`
+}
+
+type ConsistencyCheckProof struct {
```

Comment:
Yes, it is. However, @arlolra  and I have done some changes in 7e807e5821ed0b73622108b2b1b746fc3a42f8ca which maybe solved this issue.